client/src/components/profile/components/SocialIcons.js

```diff
@@ -27,31 +27,51 @@ const propTypes = {
 function LinkedInIcon(linkedIn) {
   return (
     <a href={linkedIn} rel='noopener noreferrer' target='_blank'>
-      <FontAwesomeIcon icon={faLinkedin} size='2x' />
+      <FontAwesomeIcon
+        aria-hidden='false'
+        aria-label='LinkedIn icon'
```

Comment:
Aria label should be set on a link tag and describe where it goes. Additional attributes on icon are not need.

Reply:
Done